_data/projects/docusaurus.yml

```diff
@@ -0,0 +1,14 @@
+name: Docusaurus
+desc: (Facebook Open Source) Easy to maintain open source documentation websites that powers website of React Native, Prettier, WarriorJS and Jest
+site: https://docusaurus.io
+tags:
+- javaScript
+- library
+- UI
+- react
+- documentation
+- express.js
+- static website generator
+upforgrabs:
+  name: good first contribution
```

Comment:
@endiliey Your label name and the label link are not for each other, is it fine this way?